docs/getting-started/first-service.md

```diff
@@ -1,21 +1,18 @@
 ---
 id: first-service
 title: Create Your First Service
-description: Amplication services are a full solution complete with authentication, authorization, logging, git sync, and other essential features for back-end management.
+description: Amplication services are a full back-end solution complete with authentication, authorization, logging, git sync, and other essential features for back-end management.
 sidebar_label: Create Your First Service
 slug: /first-service
 ---
 
 # Create Your First Service With Amplication
 
-Amplication accelerates your Node.js development by helping you build production-ready backends.
-These automatically-generated backends are known as **services**.
+Amplication accelerates your app development by helping you build production-ready backends in either Node.js or .NET. These automatically-generated backends, known as **services**, utilize modern frameworks such as NestJS and Prisma for Node.js, and ASP.NET Core alongside Entity Framework for .NET.
```

Comment:
@mulygottlieb - Is that accurate?

Reply:
@PazYanoverr @dericksozo yes it is accurate, except for adding "Core" also to "Entity Framework" (write "Entity Framework Core")